packages/bvaughn-architecture-demo/components/sources/PointPanel.module.css

```diff
@@ -56,8 +56,8 @@
   overflow: auto;
   height: 100%;
   flex: 1;
-  background-color: var(--point-panel-input-background-color);
-  border: 2px solid transparent;
+  background-color: var(--body-bgcolor);
```

Comment:
This variable: `-point-panel-input-background-color`

Exists specifically for this case. If you think it should be a different color, update the variable? 😄

Reply:
I think it's nice to remove variables that are unnecessary. With that said, there are several variables that i could probably kill now, but haven't :)

Reply:
I think we might disagree here a bit.

Our UI mocks often have divergences between light/dark theme that can be hard to implement if we're using building block variables (like `--body-bgcolor`). Variables like `--point-panel-input-background-color` are a way to mange that complexity/scoping. (We can't change `--body-bgcolor` for example, without breaking a lot of things.)